pipelines/common/src/main/java/com/google/fhir/analytics/DwhFiles.java

```diff
@@ -222,16 +224,24 @@ public Set<String> findNonEmptyViews(Boolean viewFlag) throws IOException {
 
     Set<String> typeSet = Sets.newHashSet();
     for (String file : fileSet) {
-      if (viewFlag) {
+      if (findResourceTypes) {
         try {
           fhirContext.getResourceType(file);
           typeSet.add(file);
         } catch (DataFormatException e) {
           log.debug("Ignoring file {} which is not a FHIR resource.", file);
         }
       } else {
+        // Logic to verify if view directory contains parquet files
```

Comment:
Just to clarify what I meant in my previous comment: The validation is similar to what we have above on line 229 (`fhirContext.getResourceType(file)`) but for views. There we are checking that the directory actually corresponds to FHIR resource; here we should check if the directory corresponds to a view.

Reply:
Because the naming convention for incremental runs is very generic ('output-x-of-x.parquet' with x being an integer) in comparison to the batch output (which contains the resource type and view name). I decided to check if the view directory contains files with the `.parquet` substring.

Reply:
I am a little bit confused by your comment; the files that we are getting here are the _directories_ containing `.parquet` files, correct (note `getCurrentDirectory` on line 217)? I mean, in case of resources (`findResourceTypes == true`) we return directories like `Patient` and `Observation`. In the case of views, we should return _directories_ whose names are view-names; does that make sense?

If yes, then all I am saying is that like line 229 above, we should validate and only return directories that correspond to view names. Actually thinking more about this, I think this is actually a bug because we will return resource dirs (e.g., `Patient`, `Observation`) as well, when `findResourceTypes` is `false`.

Reply:
Ok, I understand what you mean. Fixed this issue by adding `ViewManger` validation, and adjusted the `LocalDwhFilesTest` to reflect this.